package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mattermost-redux",
-  "version": "5.3.1",
+  "version": "5.6.0",
```

Comment:
Why are you changing the version here?

Reply:
It's something that keeps accidentally getting pulled into my rebases.